### ISSUE TITLE
Download stops before reaching limit

### ISSUE THREAD

**Issue by aggoaggo** [hardikvasa/google-images-download#294]
When trying to download a 1000 images, it always stops before this, no matter what keyword I am using, I get an error message like this:  _"Unfortunately all 1000 could not be downloaded because some images were not downloadable. 556 is all we got for this search filter!"_

I am assuming, that perhaps google image search doesn't return more results? At least not in a first default search, as all downloading seem to stop at around 500-600.  Is there any way to extend this, to get 1000 + images downloaded?

**Comment by glenn-jocher** [comment]
@aggoaggo did you find a solution to this? I believe that the search simply reaches the end of the allowable scrolling range for a google/bing image search. I found this typically occurs after around 500-600 images. I don't see any workaround.

<img width="939" alt="Screen Shot 2020-02-25 at 10 04 06 PM" src="https://user-images.githubusercontent.com/26833433/75316734-f588d200-581a-11ea-829f-3e7c02275310.png">

**Comment by aggoaggo** [comment]
Yeah, that's probably it, also get stop around those numbers... No I haven't found any solutions.

**Comment by fantasy-fish** [comment]
I changed the code around line 200, where I increased the number of loop. It seems to help a little bit, but still not solve the problem from the root. I did it because I think the reason may be the page not given long enough time to load all the images.

`times = 10
        for i in range(30*times):
            element.send_keys(Keys.PAGE_DOWN)
            time.sleep(0.3)

        try:
            browser.find_element_by_id("smb").click()
            for i in range(50*times):
                element.send_keys(Keys.PAGE_DOWN)
                time.sleep(0.3)  # bot id protection
        except:
            for i in range(10*times):
                element.send_keys(Keys.PAGE_DOWN)
                time.sleep(0.3)  # bot id protection

        print("Reached end of Page.")
        time.sleep(0.5)`
